AbletonMCP_Remote_Script/commands/browser.py

```
@@ -0,0 +1,449 @@
+        if track_index < 0 or track_index >= len(song.tracks):
+            raise IndexError("Track index out of range")
+
+        track = song.tracks[track_index]
+
+        # Find the browser item by URI
+        item = _find_browser_item_by_uri(
+            app.browser, item_uri, context.browser_uri_cache
+        )
+
+        if not item:
+            raise ValueError(f"Browser item with URI '{item_uri}' not found")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)